docs/languages/markdown.md

```diff
@@ -51,25 +51,17 @@ Additionally, double clicking an element in the Markdown preview will automatica
 
 ![Markdown Preview double click switches to editor](images/Markdown/double-click-preview-switch.gif)
 
-### Using your own CSS
-
-By default, we use a CSS style for the preview that matches the style of VS Code. If you want to use your own CSS for the Markdown preview, update the `"markdown.styles": []` [setting](/docs/getstarted/settings.md) with the comma-separated list of URL(s) for your style sheet(s). Note that `markdown.styles` files must either be served over https or point to a local file within the current workspace.
+### Extending the Markdown preview
 
-For instance, in the screen shot above we used a custom CSS that changes the default font for the page and changes the color for the H1 title.
+Extensions can contribute custom styles and scripts to the markdown preview to change its appearance and add new functionality. Here's a set of example extensions that customize the preview:
 
-Here is the relevant CSS:
+<div class="marketplace-extensions-markdown-preview-curated"></div>
```

Comment:
Not sure what the procedure is for getting a new curated extension section. Would like to have extensions like markdown-math and markdown-github-style listed here

Reply:
Here's a list of candidate extensions:

- https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles
- https://marketplace.visualstudio.com/items?itemName=hbrok.markdown-preview-bitbucket
- https://marketplace.visualstudio.com/items?itemName=goessner.mdmath
- https://marketplace.visualstudio.com/items?itemName=bierner.markdown-emoji